vue-todo/src/store/index.js

```diff
@@ -5,8 +5,24 @@ Vue.use(Vuex)
 
 export default new Vuex.Store({
   state: {
+    todos : []
+  },
+  getters: {
+    todos: state => {
+      return state.todos
+    }
   },
   mutations: {
+    ADD_TODO (state, comment) {
+      
+      const id = state.todos.length;
+
+      state.todos.push({
+        id: id,
```

Comment:
オブジェクトはkeyとvalueが一致している場合、以下のように省略記法が使用できますので、該当する箇所は全て修正してみましょう。
```suggestion
        id,
```